Formula/p/pistache.rb

```diff
@@ -0,0 +1,161 @@
+#
+# SPDX-FileCopyrightText: 2024 Duncan Greatwood
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+class Pistache < Formula
+  desc "Modern, fast, elegant HTTP + REST C++17 framework with pleasant API"
+  homepage "https://github.com/pistacheio/pistache"
+  url "https://github.com/pistacheio/pistache/archive/refs/tags/v0.4.25.tar.gz"
+  sha256 "30569c5d7290f38d2874e9a7c1a2e9a1a8f16b7567f581eba9590d7e1016756e"
+  license "Apache-2.0"
+  head "https://github.com/pistacheio/pistache.git", branch: "master"
+
+  depends_on "brotli" => :build
+  depends_on "cmake" => :build # Required so meson finds hinnant-date
+  depends_on "cpp-httplib" => :build
+  depends_on "googletest" => :build
+  depends_on "howard-hinnant-date" => :build
+  depends_on "libevent" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkgconf" => :build
+  depends_on "rapidjson" => :build
+  depends_on "zstd" => :build
+
+  def install
+    system "meson", "setup", "build",
+           "--buildtype=release",
```

Comment:
I believe this is part of the standard arguments already

Reply:
Per the cookbook, that's correct, thanks. I removed the redundant
`--buildtype=release`